### ISSUE TITLE
Delayed Match to Sample Env Example Notebook: Model Performance at Chance

### ISSUE THREAD

**Issue by sreejank** [neurogym/neurogym#46]
Hi,

I think this is a great library! Thanks so much for making it. I was interested in writing my own modification of the Delayed Match to Sample environment, and tried going through this Notebook to start: https://neurogym.github.io/envs/DelayMatchSample-v0.html. 

I saw the model performance does not so well (the training loss doesn't go down that much after 2000-10000 epochs, and the performance at test time is basically at chance). I haven't explored tuning the learning rate, etc yet. But I wanted to get your guys intuition on: how hard this environment should be for these kind of recurrent models and what kinds of architectural choices are recommended. I also saw there's an RL version and a supervised version. Are there any advantages of using the RL version? (I'd imagine that supervised is easier, but just wanted to make sure). 

Thanks in advance!

**Comment by gcroci2** [comment]
Is this still relevant? Otherwise, we can close it @nacloos @manuelmolano

**Comment by cantordust** [comment]
Did you ever resolve this issue? I don't know what model you used, but at any rate 10k epochs sounds like a really long training time for such a task.

Marking as stale for imminent closure. Can be reopened if necessary.
